extension/src/test/suite/experiments/index.test.ts

```diff
@@ -561,6 +561,34 @@ suite('Experiments Test Suite', () => {
       )
     }).timeout(WEBVIEW_TEST_TIMEOUT)
 
+    it('should be able to handle a message to rename an experiment', async () => {
+      const { mockMessageReceived } =
```

Comment:
[F] This is failing in the CI:  
```Error: Timeout of 16000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/test/suite/experiments/index.test.js)```

Reply:
is it passing locally?

Reply:
I think you'll need to stub the version of the CLI returned by setup:
```
stub(Setup.prototype,'getCliVersion').returns('3.22.0')
```

Reply:
☝🏻 this is a complete guess.

Reply:
The CI doesn't have access to DVC. Your local test environment probably does.

Reply:
sorry should be `resolves` not `returns` (tested locally)

Reply:
and there is a timing issue. Without going to deep into the inner workings of Node we need to wait for expRename to be called. We can do that like this:

```
    it('should be able to handle a message to rename an experiment', async () => {
      const { mockMessageReceived } =
        await stubWorkspaceGettersWebview(disposable)

      const mockNewExperimentName = 'new-experiment-name'
      const inputEvent = getInputBoxEvent(mockNewExperimentName)

      stub(Setup.prototype, 'getCliVersion').resolves('3.22.0')

      const mockRenameExperiment = stub(DvcExecutor.prototype, 'expRename')

      const mockRenameCalled = new Promise(resolve =>
        mockRenameExperiment.callsFake(() => {
          resolve(undefined)
          return Promise.resolve('Experiment Renamed')
        })
      )

      const mockExperimentId = 'exp-e7a67'

      mockMessageReceived.fire({
        payload: mockExperimentId,
        type: MessageFromWebviewType.RENAME_EXPERIMENT
      })

      await Promise.all([inputEvent, mockRenameCalled])
      expect(mockRenameExperiment).to.be.calledOnce
      expect(mockRenameExperiment).to.be.calledWithExactly(
        dvcDemoPath,
        mockExperimentId,
        mockNewExperimentName
      )
    }).timeout(WEBVIEW_TEST_TIMEOUT)
```